sql/table.go

```diff
@@ -164,6 +164,8 @@ func encodeTableKey(b []byte, v parser.Datum) ([]byte, error) {
 		return encoding.EncodeNumericFloat(b, float64(t)), nil
 	case parser.DString:
 		return encoding.EncodeBytes(b, []byte(t)), nil
+	case parser.DNull:
+		return b, nil
```

Comment:
Yeah, we're gonna need a sentinel. I think it'd be reasonable to track that as part of #1909.